Buildings/Resources/Data/Fluid/DXSystems/Heating/AirSource/Validation/SingleSpeedHeating_OnDemandResistiveDefrost/DXCoilSystemAuto.idf

```diff
@@ -0,0 +1,2538 @@
+!          |                    |                        |
+!          |                    |                        |
+!          |                    |                        |
+!          |       West         |         East           |    window = *
+!          |                    |                        |
+!          |                    |                        |
+!          |                    |                        |
+!          |___**************___|________________________|
+!
+!      (0,0,0)             (6.069,0,0)               (12.192,0,0)
+!
+! Internal gains description:     ZONE West      - lighting is 0 watts, equip is 2928.751 watts, 3 occupants
+! Internal gains description:     ZONE North     - lighting is 878.6252 watts, equip is 2928.751 watts, 4 occupants
+! Internal gains description:     ZONE East      - lighting is 1464.375 watts, equip is 1464.375 watts, 3 occupants
+!
+! Interzone Surfaces:             3 interzone surfaces (see diagram)
+! Internal Mass:                  None
+! People:                         10
+! Lights:                         2343 W
+! Equipment:                      7323 W
+! Windows:                        1
+! Detached Shading:               None
+! Daylight:                       None
+! Natural Ventilation:            None
+! Compact Schedules:              Yes
+! Solair Distribution:            FullInteriorAndExterior
+!
+! HVAC:                           UNITARYSYSTEM:HEATPUMP:AIRTOAIR
+!                                 Heat pump using a single air loop serving 3 zones.  Heat pump configured with a
+!                                 blow through fan, heat pump defrost is timed using a resistive heating element.
+!                                 Heating set point during winter months is 20 C during occupied hours,
+!                                 15 C set-back otherwise.  Cooling set point at 24 C during occupied hours only.
+!                                 Cooling coil off all winter. Heat Pump has a gas supplemental heater.
+!
+! Zonal Equipment:                DIRECT AIR
+! Central Air Handling Equipment: Yes, AirLoopHVAC:HeatPumpOnly, AirLoopHVAC:UnitaryCoolOnly
+! System Equipment Autosize:      Yes
+! Purchased Cooling:              No
+! Purchased Heating:              No
+! Purchased Chilled Water:        No
+! Purchased Hot Water:            No
+! Coils:                          Coil:Cooling:DX:SingleSpeed, Coil:Heating:DX:SingleSpeed, Coil:Heating:Fuel
+! Pumps:                          None
+! Boilers:                        None
+! Chillers:                       None
+! Towers:                         None
+!
+! Results:
+! Standard Reports:               All Summary Predefined Reports
+! Timestep or Hourly Variables:   TimeStep
+! Time bins Report:               None
+! HTML Report:                    Predefined All Summary
+! Environmental Emissions:        None
+! Utility Tariffs:                None
+Version,
+    22.2;                    !- Version Identifier
```

Comment:
@karthikeyad-pnnl : Can you please update the two new idf files that you added to EnergyPlus 23.1.
We update the CI testing to EnergyPlus 23.1 through https://github.com/lbl-srg/modelica-buildings/pull/3427, which also updates the existing idf files, but it does not contain your new idf files.